<launch>
  <arg name="video_stream_url" default="$(env ROCON_RTSP_CAMERA_RELAY_URL)"/>
  <node pkg="rocon_rtsp_camera_relay" name="rtsp_camera_relay" type="rocon_rtsp_camera_relay_node">
    <param name="video_stream_url"    value="$(arg video_stream_url)"/>
    <param name="camera_name" value="overhead_cam"/>
    <param name="camera_config_url" value="file:///$(find camera_cal)/overhead_camera_calibration.yaml"/>
  </node>

  <node name="rectifier" pkg="image_proc" type="image_proc" ns="overhead_cam" respawn="true"/>
</launch>
